recipes/libgcrypt/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
+
+make -j4
```

Comment:
Don't hardcode to `-j4`. Use the variable `$CPU_COUNT` instead. (I don't think we have 4 cores in any of the CIs BTW :smile: )